src/interfaces/python_modular/CMakeLists.txt

```diff
@@ -66,4 +66,4 @@ configure_file(${CMAKE_CURRENT_BINARY_DIR}/main_init.py.in main_init.py COPYONLY
 INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/main_init.py
 	DESTINATION ${PYTHON_PACKAGES_PATH}/shogun
   COMPONENT python
-	RENAME __init__.py)
+	RENAME __init__.py)
```

Comment:
I had generated interface code and accidentally committed it, so I removed all the files from that directory, restored the ones from the head (that were there by default) and re-commited them.  I forgot to restore this file, and after struggling to get it from the remote repository, I just recreated it and did a cut and paste.

**tdlr** Still learning about git.